src/aeso_ast_to_fcode.erl

```diff
@@ -363,7 +363,7 @@ to_fcode(Env, [{Contract, Attrs, Con = {con, _, Name}, Decls}|Rest])
             to_fcode(Env1, Rest)
     end;
 to_fcode(_Env, [NotMain = {NotMainHead, _ ,_ , _}]) when NotMainHead =/= contract_def ->
-    fcode_error({last_declaration_must_be_contract_def, NotMain});
+    fcode_error({last_declaration_must_be_main_contract, NotMain});
```

Comment:
How was this working before you made this change? Apparently there was no handling of `last_declaration_must_be_contract_def` error. Shouldn't be a test somewhere for that?

Reply:
This is an fcode generation error, currently it is impossible for a user to cause it. It is left for cases where someone pins in the middle of the compilation pipeline, or something goes bad in the typechecking process